src/it/java/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/WhitespaceAroundTest.java

```diff
@@ -64,7 +64,6 @@ public void testWhitespaceAroundBasic() throws Exception {
             "150:20: " + getCheckMessage(messages, msgPreceded, ":"),
             "249:14: " + getCheckMessage(messages, msgPreceded, "->"),
             "250:15: " + getCheckMessage(messages, msgFollowed, "->"),
-            "250:17: " + getCheckMessage(messages, msgPreceded, "{"),
```

Comment:
We should add to our Inputs a case that was in issue.
`l = () -> {}; // ok` and clearly declare that we support such code